src/@utils/accessDetailsAndPricing.ts

```diff
@@ -256,6 +256,7 @@ export async function getOrderPriceAndFees(
     asset?.services[0].serviceEndpoint
   )
   orderPriceAndFee.providerFee = initializeData.providerFee
+  if (!orderPriceAndFee.providerFee) return
```

Comment:
double checking: under what circumstances could this `orderPriceAndFee.providerFee` be `null` or `undefined`? As this line stops all price fetching in case this is not defined. So question is if we should if maybe somewhere the provider owner would have not set it or something. Do we have a default set wherever we get this from which is not  `null` or `undefined`?

Reply:
This is a edge case when the network is slow. It doesn't happen a lot, but basically yesterday I encounter an issue where the query failed (because infura was slow/down, today i haven't had this issue) and this failing call makes [this line](https://github.com/oceanprotocol/market/blob/fix/issue-1356-loader-stuck-button-asset/src/@utils/accessDetailsAndPricing.ts#L289) of code throw an error. To avoid this extremely limited edge case, I added this return when providerFee is null (aka there's an error retrieving the data). We can remove it, if we think this will not create a problem in the future.